source/services/encryption/DeviceLocalAESStrategy.ts

```diff
@@ -0,0 +1,68 @@
+import { NativeModules } from "react-native";
+import {
+  EncryptionContext,
+  EncryptionDependencies,
+  EncryptionPossibility,
+  IEncryptionStrategy,
+} from "./EncryptionStrategy";
+
+const { Aes } = NativeModules;
+
+export interface DeviceLocalAESParams {
+  key: string;
+  iv: string;
+}
+
+async function createAesParams(): Promise<DeviceLocalAESParams> {
+  const randomBytes = await Aes.randomKey(16);
+
+  const key = await Aes.pbkdf2(randomBytes, "salt4D42bf960Sm1", 5000, 256);
```

Comment:
I suggest that values like `"salt4D42bf960Sm1", 5000, 256` are put in constants for readability